src/itti/itti.hpp

```diff
@@ -240,7 +240,7 @@ class itti_mw {
    *  \param task_id task id of the task that requested the timer
    *  @returns -1 on failure, 0 otherwise
    **/
-  int timer_remove(timer_id_t& timer_id);
+  int timer_remove(const timer_id_t& timer_id);
```

Comment:
@lionelgo 

what is the purpose of having a `const` on your timer_id? I am a bit rusty on my C/C++.

Reply:
Cv-qualified references are ill-formed except when the cv-qualifiers are introduced through the use of a typedef-name or decltype-specifier, in which case the cv-qualifiers are ignored.

Would have been a good idea to use const on a timer_id_t, not a good idea to use it on a timer_id_t&